app/controllers/HomepageController.scala

```diff
@@ -39,7 +46,7 @@ class HomepageController @Inject() (val controllerComponents: SecurityComponents
   def judgmentHomepage(): Action[AnyContent] = secureAction { implicit request: Request[AnyContent] =>
     {
       if (request.token.isJudgmentUser) {
-        Ok(views.html.judgment.judgmentHomepage(request.token.name))
+        Ok(views.html.judgment.judgmentHomepage(request.token.name, blockViewTransfers))
```

Comment:
I'm not sure we're releasing this functionality for judgment users, so could you just default this to `true` for the judgment template, until this is decided.